shinydashboard/lantern/config.yml

```diff
@@ -0,0 +1,8 @@
+default:
+  lantern:
+    driver: 'Postgres' 
+    server: 'host.docker.internal'
+    uid: 'lantern'
+    pwd: 'postgrespassword'  
```

Comment:
is there a way we can hide this in the future? we currently store config variables as environment variables as much as possible, which makes them easy to set with docker. we still have security concerns around this that we plan to work out with the security team.

Reply:
Yes, working on passing environment variables to the shiny docker container.